src/views/Farms/components/FarmCard/ApyCalculatorModal.tsx

```diff
@@ -50,17 +50,17 @@ const ApyCalculatorModal: React.FC<ApyCalculatorModalProps> = ({
       <Grid>
         <GridItem>
           <Text fontSize="12px" bold color="textSubtle" textTransform="uppercase" mb="20px">
-            {TranslateString(999, 'Timeframe')}
+            {TranslateString(860, 'Timeframe')}
           </Text>
         </GridItem>
         <GridItem>
           <Text fontSize="12px" bold color="textSubtle" textTransform="uppercase" mb="20px">
-            {TranslateString(999, 'ROI')}
+            {TranslateString(858, 'ROI')}
           </Text>
         </GridItem>
         <GridItem>
           <Text fontSize="12px" bold color="textSubtle" textTransform="uppercase" mb="20px">
-            {TranslateString(999, 'CAKE per $1000')}
+            {TranslateString(864, 'CAKE per $1000')}
```

Comment:
This is fine for now but any translations with dynamic values should be like this - `CAKE per %amount%`